evil-ediff.el

```diff
@@ -139,7 +166,11 @@
     ("\C-z" . ediff-suspend)
     ("z"    . nil)
     ("zl"   . evil-ediff-scroll-right)
-    ("zh"   . evil-ediff-scroll-left))
+    ("zh"   . evil-ediff-scroll-left)
+    ("b"    . evil-ediff-copy-B-to-A)
+    ("a"    . evil-ediff-copy-A-to-B)
```

Comment:
my mistake,
there should be `h` for evil-ediff-copy-B-to-A
and `l` for  evil-ediff-copy-A-to-B